ui/src/reducers/tests/draftItem.test.js

```diff
@@ -0,0 +1,218 @@
+  actionsLayerType: null,
+  showPreviewer: false,
+  filePreviewEditLayer: true,
+  filePreviewEdit: {},
+  uploadFiles: Map({}),
+  pathSelected: null,
+
+  workflows: fromJS([]),
+  workflows_items: fromJS({}),
+  bucket: Map({}),
+  formData: null,
+  // From backend: deposit resource
+  access: null,
+  can_admin: false,
+  can_update: false,
+  created: null,
+  loading: false,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {},
+  revision: null,
+  schema: null,
+  schemas: null,
+  status: null,
+  type: null,
+  updated: null
+});
+const generalTitleStateBefore = Map({
+  errors: [],
+  schemaErrors: [],
+  actionsLayer: false,
+  actionsLayerType: null,
+  showPreviewer: false,
+  filePreviewEditLayer: true,
+  filePreviewEdit: {},
+  uploadFiles: Map({}),
+  pathSelected: null,
+
+  workflows: fromJS([]),
+  workflows_items: fromJS({}),
+  bucket: Map({}),
+  formData: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
+    }
+  },
+  // From backend: deposit resource
+  access: null,
+  can_admin: false,
+  can_update: false,
+  created: null,
+  loading: false,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is an old Title"
+    }
+  },
+  revision: null,
+  schema: null,
+  schemas: null,
+  status: null,
+  type: null,
+  updated: null
+});
+const generalTitleStateAfter = Map({
+  errors: [],
+  schemaErrors: [],
+  actionsLayer: false,
+  actionsLayerType: null,
+  showPreviewer: false,
+  filePreviewEditLayer: true,
+  filePreviewEdit: {},
+  uploadFiles: Map({}),
+  pathSelected: null,
+
+  workflows: fromJS([]),
+  workflows_items: fromJS({}),
+  bucket: Map({}),
+  formData: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
+    }
+  },
+  // From backend: deposit resource
+  access: null,
+  can_admin: false,
+  can_update: false,
+  created: null,
+  loading: false,
+  created_by: null,
+  experiment: null,
+  files: Map({}),
+  id: null,
+  links: null,
+  metadata: {
+    additional_resources: {
+      internal_discussions: "blah blah"
+    },
+    basic_info: {
+      abstract: "This is the abstract",
+      conclusion: "This is the conclusion",
+      ana_notes: ["AN-2121/212"],
+      general_title: "This is the new title"
```

Comment:
`general_title` correct position is under `metadata` ( `metadata > general_title` )

Reply:
fixed